core/roboconf-dm-monitoring/src/main/java/net/roboconf/dm/monitoring/MonitoringService.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2013-2015 Linagora, Université Joseph Fourier, Floralis
+ *
+ * The present code is developed in the scope of the joint LINAGORA -
+ * Université Joseph Fourier - Floralis research program and is designated
+ * as a "Result" pursuant to the terms and conditions of the LINAGORA
+ * - Université Joseph Fourier - Floralis research program. Each copyright
+ * holder of Results enumerated here above fully & independently holds complete
+ * ownership of the complete Intellectual Property rights applicable to the whole
+ * of said Results, and may freely exploit it in any manner which does not infringe
+ * the moral rights of the other copyright holders.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.roboconf.dm.monitoring;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Set;
+
+import net.roboconf.core.model.beans.Application;
+
+/**
+ * Interface for monitoring template management.
+ *
+ * @author Pierre Bourret - Université Joseph Fourier
+ */
+public interface MonitoringService {
+
+	/**
+	 * The name of the directory where application monitoring templates are read from, relative to the Roboconf
+	 * configuration directory.
+	 */
+	String MONITORING_TEMPLATE_DIRECTORY = "monitoring-template";
+
+	/**
+	 * The name of the directory where application monitoring generated files are written/updated, relative to the
+	 * Roboconf configuration directory.
+	 */
+	String MONITORING_TARGET_DIRECTORY = "monitoring-generated";
+
+	/**
+	 * The extension of monitoring template files.
+	 */
+	String MONITORING_TEMPLATE_FILE_EXTENSION = ".tpl";
+
+	/**
+	 * Get the target directory, where monitoring reports are generated.
```

Comment:
According to Oracle's guidelines about writing Javadoc, methods documentation should use the third person.
Here...

> Gets the...